normandy/conftest.py

```diff
@@ -33,3 +35,19 @@ def geolocation():
         skip_except_in_ci()
     else:
         return geolocation_module
+
+
+@pytest.fixture
+def mocked_autograph(mocker):
+    mocked = mocker.patch('normandy.recipes.models.Autographer')
+
+    def fake_sign(datas):
+        sigs = []
+        for d in datas:
+            sigs.append({
+                'signature': hashlib.sha256(d).hexdigest()
```

Comment:
You're using `sha256` in enough places that I wonder if you should factor it to a common place in case we change how signatures are generated for testing in the future? I'm not entire sure about that though.